### ISSUE TITLE
[Bug] Blurry icons on start tiles

### ISSUE THREAD

**Issue by mah-qurashy** [JasonStein/Notepads#499]
I noticed the icon on the start tile is a bit blurry (not much though)
![tile100](https://user-images.githubusercontent.com/20822445/81280259-7e608d00-9058-11ea-8cb4-76c5898de697.png)
as i investigated this, the tile dimensions at 100% scaling is 100x100 pixels while the resource provided for this `Square150x150Logo.scale-100.png` is 150x150 pixels, so the image is resized.
i looked into other uwp apps, they seem to have the same issue, the tile resource is 150x150 too.
the resizing algorithm that windows is using seems to be good, but not as good as providing an appropriately sized resource.
any thoughts if this is fixable? or why do we use 150x150 image? could we simply make it 100x100 or would that break something else?

**Comment by mah-qurashy** [comment]
also i looked into full screen start menu, it has 124x124 tiles so it is resized from the 150x150 image as well
![image](https://user-images.githubusercontent.com/20822445/81281778-92a58980-905a-11ea-9888-6c369e85a768.png)
could it be that windows has somewhere else where it uses a 150x150 tile?

**Comment by mah-qurashy** [comment]
also classic win32 apps don't have this issue since they simply use the 32x32 icon on the tile, no tile resource provided

**Comment by JasonStein** [comment]
I will take a look.

**Comment by JasonStein** [comment]
@mah-qurashy I don't think we can just delete them. Maybe you can take a look at this one: https://github.com/UWPCommunity/Quick-Pad/tree/master/src/QuickPad.UI/QuickPad.UI/Assets

@yaichenbaum

**Comment by yaichenbaum** [comment]
@JasonStein I actually used Visual Asset Generator to create the icons for Quick Pad, I remember spending a lot of time on it and it took some time to get it right. I will see if I can get Robby who has spent a lot of time on this area to join this thread.

**Comment by RobbyRobbyRobby** [comment]
Hi, 

There could be many reasons for this, to look into it I recommend downloading Visual Asset Generator (it's 100% free, no ads or accounts, etc) and try the different resamplers.

I think it is likely that you want a harsher resample for smaller icons.  The softer ones can leave smaller pictures looking blurry.

If nothing else, this will give us an idea of where the issue may be.

Cheers,
Rob.

**Comment by mah-qurashy** [comment]
@JasonStein do you think creating a `Square150x150Logo.targetsize-100.png` could work?

**Comment by JasonStein** [comment]
> @JasonStein do you think creating a `Square150x150Logo.targetsize-100.png` could work?

You can give it a try, I am trying to create some icons using the app Robby created to see if it works:
![image](https://user-images.githubusercontent.com/1166162/81493401-e976b300-9254-11ea-8ec9-3b48d1784b75.png)

**Comment by JasonStein** [comment]
> @JasonStein do you think creating a `Square150x150Logo.targetsize-100.png` could work?

No luck, maybe we should adjust pixel size of the icon to see if it works?

**Comment by mah-qurashy** [comment]
> No luck, maybe we should adjust pixel size of the icon to see if it works?

I'm sure that this would work, what concerns me is that it might make somewhere else look even more blurry
i mean even the design documentations clearly state that medium tiles are 150x150
![image](https://user-images.githubusercontent.com/20822445/81495268-79494b00-92af-11ea-9205-a73bdc8b38af.png)
for now as I'm unable to find a place in windows that uses a 150x150 tile, the obvious drawback of resizing is that the tile in full screen start menu will look more blurry (will be upscaled from 100x100 instead of downscaled from 150x150)
